main.tf

```diff
@@ -104,14 +104,15 @@ module "redis" {
 }
 
 locals {
-  redis_connection_string = var.create_redis ? "redis://${module.redis.0.connection_string}?tls=true&ttlInSeconds=60" : null
+  redis_certificate       = var.create_redis ? module.redis.0.ca_cert : null
+  redis_connection_string = var.create_redis ? "redis://:${module.redis.0.auth_string}@${module.redis.0.connection_string}?tls=true&ttlInSeconds=60&caCertPath=/tmp/server_ca.pem" : null
```

Comment:
Should the caCertPath be customizable or at least defaulting to a standard location such as `/etc/ssl/certs/` ?

Reply:
Is that path more common? I was going by google's docs here https://cloud.google.com/memorystore/docs/redis/enabling-in-transit-encryption. Why would customers want to customize this?

Reply:
It's the default for debian-based distributions (e.g. in Ubuntu https://ubuntu.com/server/docs/security-certificates#:~:text=The%20default%20location%20to%20install%20certificates%20is%20/etc/ssl/certs). It's something that is usually more standard/restricted than /tmp which could be blown away easily.

Reply:
I wonder why they suggested tmp?

Reply:
Get up and running quickly is my guess? I can ask Google reps tomorrow in our weekly sync to find out.

Reply:
ok, well not a huge deal, I'll update to the more standard location